app/src/main/java/org/p2p/wallet/send/interactor/SearchInteractor.kt

```diff
@@ -89,7 +85,6 @@ class SearchInteractor(
                     ),
                     canReceiveAndBuy = true
                 )
-                hasEmptyBalance -> SearchResult.EmptyBalance(addressState)
```

Comment:
should we remove this class at all then?

Reply:
we can but.... do we have 100% clarity that we do not need this at all?